static/scss/detail/courses-in-program.scss

```diff
@@ -94,23 +94,26 @@
     font-weight: 500;
     margin: 0 0 20px;
     line-height: 30px;
-    color: black;
   }
 
   p {
     margin: 0 0 10px;
     min-height: 89px;
+    padding-bottom: 40px;
+    padding-top: 20px;
+    text-align: start;
   }
 
   .read-more {
```

Comment:
(Optional) If we can rename this class name to match the text, as for now text is changed to `View details`

Reply:
Updated to `detail-link`